src/index.js

```diff
@@ -0,0 +1,70 @@
+import './styles.css';
+import {
+  apiGet, sendLike, getLikes, involvementApiGet, involvementApiPost,
+} from './modules/api.js';
+import {
+  generateMovies, generatePopup, generateComments, generateNoComments,
+} from './modules/userInteraction.js';
+import episodesCounter from './modules/episodesCounter.js';
+import commentCounter from './modules/commentsCounter.js';
+
+const mainBody = document.getElementById('main');
+const popup = document.getElementById('popup-window');
+
+window.addEventListener('load', async () => {
+  const numberOfLikes = await getLikes();
+  const epiArray = await apiGet();
+  for (let i = 0; i < epiArray.length; i += 1) {
+    if (epiArray[i].season === 1) {
+      mainBody.innerHTML += generateMovies(epiArray[i], numberOfLikes[i].likes);
```

Comment:
- [ ] As requested by the previous code reviewer,  It would be better if you make the like icons show as we have in the mockup or [design guidelines](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). Currently, the icons are not showing as you can see below :point_down: 

  ![image](https://user-images.githubusercontent.com/92303891/233813277-e3f51bc7-5073-4124-ae6d-e9e0a7821226.png)
  
  ![image](https://user-images.githubusercontent.com/92303891/233813301-47ae72ea-ee7a-4cef-9a2c-13bb17238354.png)